core/app/models/workarea/pricing/sku.rb

```diff
@@ -47,14 +47,22 @@ def find_price(options = {})
       end
 
       # Default selling price, when one unit is placed
-      # into the cart of an anonymous user.
+      # into the cart of an anonymous user. Determines whether the SKU
+      # is on sale and returns the `:sale` price if so, otherwise it
+      # will return the `:sell` price.
       #
       # @return [Money]
       #
       def sell_price
         find_price(quantity: 1).sell
       end
-      alias_method :regular_price, :sell_price
```

Comment:
Isn't this a breaking change? Feels like this could really break something in a build. If this is a display issue, it would sound like we could fix it without changing the definition of this one method.